float32 threshold
---
int16 status